eng/Generate.ps1

```diff
@@ -255,7 +255,8 @@ if (!($Exclude -contains "Samples"))
 $cadlRanchProjectDirectory = Join-Path $repoRoot 'test' 'CadlRanchProjects'
 $cadlRanchProjectPathsWithoutConvenience = # Needs justification to add item
     'enums/extensible', # https://github.com/Azure/autorest.csharp/issues/3079
-    'hello' # https://github.com/Azure/autorest.csharp/issues/3110
+    'hello', # https://github.com/Azure/autorest.csharp/issues/3110
+    'collectionFormat'
```

Comment:
This one has changed to return model, so we could generate convenience method.

Reply:
We have a defect to generate convenience method https://github.com/Azure/autorest.csharp/issues/3194. We will resolve this first.